### PR TITLE
Added Json format option

### PR DESCRIPTION
**Pull request by ronish31**
# What changed
See preview: https://deploy-preview-1075--logz-docs.netlify.app/shipping/log-sources/dotnet.html#nlog-config

Added Json format option t:
nlog - xml & code logger
log4net - xml & code logger

----

## Pages to review

5872532
## Remaining work

- [ ] Technical review
- [ ] Copy Review
- [ ] Redirects: 

## Post launch

To be completed by the docs team upon merge:

- [ ] Teams to update with the new information:
- [ ] Replace original content on the support portal with 'this page has been moved to ...' - paste the URL
- [ ] Update these log shipping pages in the app: - paste the URL

### CODE REVIEW

File: _source/logzio_collections/_log-sources/dotnet.md [hunk 1]
```diff
@@ -237,7 +241,9 @@ For a complete list of options, see the configuration parameters below the code
       bufferTimeout="00:00:05"
       retriesMaxAttempts="3"
       retriesInterval="00:00:02"
-      debug="false">
+      debug="false"
+      <!-- enableJson="true"-->
```
Comment (YajB) [inline _source/logzio_collections/_log-sources/dotnet.md:245]: This comment syntax does not show up as a comment (line 245)

**Review by YajB — COMMENTED** [review]
Fix syntax for the comment on line 245

**Comment by ronish31** [comment]
@YajB 245 line still doesn't appear as comment.
Also please approve the changes (changed the field's name)

**Review by YajB — APPROVED** [review]

File: _source/logzio_collections/_log-sources/dotnet.md [hunk 2]
```diff
@@ -237,7 +241,9 @@ For a complete list of options, see the configuration parameters below the code
       bufferTimeout="00:00:05"
       retriesMaxAttempts="3"
       retriesInterval="00:00:02"
-      debug="false">
+      debug="false" 
+     >
```
Comment (YajB) [inline _source/logzio_collections/_log-sources/dotnet.md:245]: Syntax misbehaves if 2nd comment is placed within the section: 
```
    <!-- Replace these parameters with your configuration -->
    <target name="logzio" type="Logzio"
      token="<<LOG-SHIPPING-TOKEN>>"
      logzioType="nlog"
      listenerUrl="https://<<LISTENER-HOST>>:8071"
      bufferSize="100"
      bufferTimeout="00:00:05"
      retriesMaxAttempts="3"
      retriesInterval="00:00:02"
      debug="false" 
      <!-- parseJsonMessage="true" -->
>  
```

**Review by YajB — APPROVED** [review]

**Pull request merged**